<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conf</class>
 <widget class="QWidget" name="Conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>564</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>564</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuración</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="8" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Paquetes inválidos</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ICMP Redirect</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Conexiones entrantes</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TCP Flooding</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TCP_SYN Erroneo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ICMP Flooding</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>288</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>288</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PARÁMETROS DEL FIREWALL</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_12">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SSH Brute force</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_10">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UDP entrantes</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_11">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UDP salientes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_8">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TCP con flags inválidos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_9">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UDP Flooding</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="minimumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>286</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Conexiones salientes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_15">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Excepto puerto 443 [HTTPS]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_32">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_35">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_13">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Excepto puerto 22 [SSH]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_36">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_37">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_14">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>216</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Excepto puerto 80 [HTTP]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Interfaz de salida</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Interfaz de entrada</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP del servidor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>288</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>288</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BLOQUEAR</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GRABAR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CANCELAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_13</tabstop>
  <tabstop>checkBox_14</tabstop>
  <tabstop>checkBox_15</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>checkBox_12</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
